lib/datasource/index.ts

```diff
@@ -208,6 +208,20 @@ export function getDefaultVersioning(datasourceName: string): string {
   return datasource.defaultVersioning || 'semver';
 }
 
+function applyReplacements(
+  dep: ReleaseResult,
+  config: GetReleasesInternalConfig
+): ReleaseResult {
+  if (config.replacementName && config.replacementVersion) {
+    return {
+      ...dep,
+      replacementName: config.replacementName,
+      replacementVersion: config.replacementVersion,
+    };
+  }
```

Comment:
Datasource is just passing through the name and version? If so, is it needed?

Reply:
Good point. I'll update the method signature and just spread the object returned from `applyReplacements`.